srcs/image/key_checks.c

```diff
@@ -0,0 +1,33 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   key_checks.c                                       :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: elouisia <elouisia@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/09/28 16:32:30 by elouisia          #+#    #+#             */
+/*   Updated: 2022/10/05 12:08:25 by elouisia         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cub3d.h"
+
+int	ft_key_check(int key, t_cub_data *data)
```

Comment:
FYI I am working on a total revamp of this function.

Reply:
Good on ya mate !